.github/workflows/release.yaml

```diff
@@ -80,9 +79,42 @@ jobs:
         if: always()
         uses: ./.github/actions/save-logs
 
-  push:
+  uds-core-test:
     runs-on: ubuntu-latest
     needs: validate
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: Download build artifacts
+        uses: actions/download-artifact@87c55149d96e628cc2ef7e6fc2aab372015aec85 # v4.1.3
+        with:
+          name: build-artifacts
+          path: build/
+
+      - name: Make UDS-CLI executable
+        run: |
+          chmod +x build/uds
+
+      - name: Deploy UDS Core bundle
+        # renovate: datasource=github-tags depName=defenseunicorns/uds-core versioning=semver
+        run: build/uds deploy ghcr.io/defenseunicorns/packages/uds/bundles/k3d-core-istio-dev:0.13.1 --confirm --no-progress
+        shell: bash
+
+      - name: Validate UDS Core deployment
+        run: |
+          build/uds zarf tools wait-for gateway admin-gateway -n istio-admin-gateway --timeout 10s
+          build/uds zarf tools wait-for gateway passthrough-gateway -n istio-passthrough-gateway --timeout 10s
+          build/uds zarf tools wait-for gateway tenant-gateway -n istio-tenant-gateway --timeout 10s
+        shell: bash
```

Comment:
`wait-for` does not need a condition like `ready` for gateways.  Below is a command without `ready` on a gateway that exists.
```
uds zarf tools wait-for gateway passthrough-gateway -n istio-passthrough-gateway --timeout 20s                                   130 ↵

 NOTE  Using config file

 NOTE  Using config file
  ✔  Waiting for gateway/passthrough-gateway in namespace istio-passthrough-gateway to exist.
```
Next is the command with `ready` on that same gateway.
```
💻 ~ » uds zarf tools wait-for gateway passthrough-gateway ready -n istio-passthrough-gateway --timeout 20s

 NOTE  Using config file

 NOTE  Using config file
     ERROR:  Wait timed out.
```
Next is the command on a gateway that doesn't exist without `ready`:
```
💻 ~ » uds zarf tools wait-for gateway fake-gateway -n istio-passthrough-gateway --timeout 20s                                            1 ↵

 NOTE  Using config file

 NOTE  Using config file
     ERROR:  Wait timed out.
```